r/reporting/reporting_ubi_8_7.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package          : reporting
+# Version          : 2.8.0.0
+# Source repo      : https://github.com/opensearch-project/reporting
+# Tested on        : UBI 8.7
+# Language         : Java
+# Travis-Check     : True
+# Script License   : Apache License, Version 2 or later
+# Maintainer       : Sunidhi Gaonkar<Sunidhi.Gaonkar@ibm.com>
+#
+# Disclaimer       : This script has been tested in root mode on given
+# ==========         platform using the mentioned version of the package.
+#                    It may not work as expected with newer versions of the
+#                    package and/or distribution. In such case, please
+#                    contact "Maintainer" of this script.
+#
+# ---------------------------------------------------------------------------
+
+yum install -y  git gcc patch sudo make java-11-openjdk-devel python39 python39-devel bzip2-devel zlib-devel openssl-devel
+
+export PACKAGE_NAME=reporting
+export PACKAGE_URL=https://github.com/opensearch-project/reporting
+export PACKAGE_VERSION=${1:-2.8.0.0}
+
+useradd -G root -d /home/testuser testuser -p test123
+sudo -E -u testuser bash -c 'bash << EOF
```

Comment:
@Sunidhi-Gaonkar1  I assume you need to be non root user to run this hence you are doing this way. 
But I will suggest to make "**use_non_root_user"= true** in build_info.json, and explicitly add sudo for yum command in build script and then you can make whole script run as normal user locally as well in currency setup.

Reply:
Sure, I will make the required changes.